src/quo2/foundations/typography.cljs

```diff
@@ -19,8 +19,7 @@
 
 (def label {:font-size   11
             :line-height 15.62
-            :letter-spacing 0.32
-            :text-transform :uppercase})
```

Comment:
I am not sure if we need label to be uppercase by default? ([figma](https://www.figma.com/file/v98g9ZiaSHYUdKWrbFg9eM/Mobile-Foundations?node-id=625%3A38))

Reply:
at least i was when i implementing it